web/cobrands/fixmystreet/admin.js

```diff
@@ -139,6 +139,20 @@ $(function(){
         return true;
     });
 
+    $(".js-group-item-add").on("click", function(e) {
+        e.preventDefault();
+        var $template_item = $(".js-group-item-template");
+        var $new_item = $template_item.clone();
+        $new_item.removeClass("hidden-js js-group-item-template");
+        $new_item.show();
```

Comment:
I don't think you need this line, or therefore also the next removeAttr line. You've already removed the hidden class, so then the insertBefore adds it to the document and there it is.